core/tests/swfs/avm1/as_broadcaster/output.txt

```diff
@@ -0,0 +1,13 @@
+true
+true
+Listener1 got the event!
```

Comment:
Can you make an assertion here about what `this` is? It's important behaviour and I think it's correct, but we aren't testing it

Reply:
What should I compare `this` to?

Reply:
`this == listener1` should work